### ISSUE TITLE
Abyss has no graph outputs

### ISSUE THREAD

**Issue by Micropathology** [galaxyproject/tools-iuc#1775]
Cannot assess assemblies in bandage could we have the .gfa outputs?

**Comment by mvdbeek** [comment]
We don't have a datatype for gfa files yet, but these are important outputs, we'll work on this (unless you want to five it a try ?)

**Comment by Micropathology** [comment]
Are there problems with handling it as a text file? I guess there are no galaxy tools that can process/use this file type at the moment so the user would have to download it anyway.

**Comment by mvdbeek** [comment]
No, but we can make GFA a subclass of text for the time being, that should help discriminate correct input/output datasets.

**Comment by Micropathology** [comment]
OK, more than willing to contribute when I will be more helpful than dangerous.

**Comment by bernt-matthias** [comment]
@Micropathology Was there any progress on this one?

**Comment by bernt-matthias** [comment]
Just for my own reference the specs: 

https://github.com/GFA-spec/GFA-spec/blob/master/GFA1.md
https://github.com/GFA-spec/GFA-spec/blob/master/GFA2.md

**Comment by bernt-matthias** [comment]
@Micropathology there are a lot of gfa outputs. Which is relevant? Or all?

E.g. there is abyss-1.gfa .. abyss-8.gfa and two symlinks abyss-contigs.gfa and abyss-scaffolds.gfa. Seems that not all of them are generated all the time.
